shells/lib/utils.js

```diff
@@ -28,7 +28,7 @@ const createPathMap = root => ({
     root,
     path: '/particles/',
     buildDir: '/bazel-bin',
-    buildOutputRegex: /\.wasm$/,
+    buildOutputRegex: '\\.wasm$',
```

Comment:
Fwiw, if the backslash-escaping is annoying, you can also do `/\.wasm$/.source` to extract a string version.

Reply:
I think it's fine.